certified-connectors/DocuSignDemo/script.csx

```diff
@@ -1639,21 +1639,27 @@ public class Script : ScriptBase
     {
       var body = ParseContentAsJObject(await response.Content.ReadAsStringAsync().ConfigureAwait(false), false);
       var query = HttpUtility.ParseQueryString(this.Context.Request.RequestUri.Query);
-      var tabType = query.Get("tabLabel");
+      var tabLabel = query.Get("tabLabel");
```

Comment:
HttpUtility.ParseQueryString returns a NameValueCollection of type String key and String value.
So, a query.Get call returns a String type.